testlib/src/test/java/io/spine/testing/logging/MemoizingStreamTest.java

```diff
@@ -37,79 +40,125 @@
 
     private static final byte[] EMPTY_BYTES = {};
 
+    private MemoizingStream stream;
+
+    @BeforeEach
+    void createStream() {
+        stream = new MemoizingStream();
+    }
+
+    @AfterEach
+    void closeStream() throws IOException {
+        stream.close();
+    }
+
+    @Nested
+    @DisplayName("provide size of the stream")
+    class Size {
+
+        private int size;
+        private byte[] input;
+
+        @BeforeEach
+        void generateInput() {
+            size = random(1000);
+            input = randomBytes(size);
+        }
+
+        @Test
+        @DisplayName("equal to zero")
+        void nothingWritten() {
+            assertThat(stream.size()).isEqualTo(0);
```

Comment:
Should we put `isEqualTo` to the next line here as well? Do we actually care about such things in tests?